testing/testing.go

```diff
@@ -346,32 +346,42 @@ func (e *TestCase) runExecution(ctx context.Context, platform *Platform) error {
 		receivedErr = res.Error
 	}
 
+	expectsErr := false
+	expectsErrText := ""
```

Comment:
To maintain code style as observed in this test func(see L342 above), consider: 
```suggestion
	var expectsErr bool
	var expectsErrText string
```

Reply:
Agree. Don't unnecessarily declare with `:=` when Go's zero-value initialization is part of the spec.  Consistency is the other consideration.

Reply:
Thanks. Fixed.